code/modules/integrated_electronics/core/printer.dm

```diff
@@ -262,7 +263,7 @@
 			if("load")
 				if(cloning)
 					return
-				var/input = capped_multiline_input(usr, "Put your code there:", "loading")
+				var/input = capped_multiline_input(usr, "Put your code there:", "loading", null, MAX_SIZE_CIRCUIT)
```

Comment:
Nulling over to the correct positional argument is really dumb. Just do `max_length=MAX_SIZE_CIRCUIT`. Keyword args exist for a reason